crates/ruff_server/src/server/api/notifications/did_change_watched_files.rs

```diff
@@ -19,7 +19,7 @@ impl super::SyncNotificationHandler for DidChangeWatchedFiles {
     ) -> Result<()> {
         for change in params.changes {
             session
-                .reload_configuration(&change.uri)
+                .reload_ruff_settings(&change.uri)
```

Comment:
nit: do we have any other kind of settings? If not, I'd drop the "ruff" word here to make it `reload_settings`